public/js/home.js

```diff
@@ -1,10 +1,11 @@
+/* global document */
+/* global window */
```

Comment:
these two lines can be one.
```/* global document window */```

Reply:
also use global fetch

Reply:
and global alert.

btw alert is generally considered a bad practice. The function alert is regarded as bad user experience, and is generally used in development for debugging purposes. You should use other methods in future projects. You don't need to change the alert now, this is just information to keep in mind in  your next project.